test/fixtures/hoc/actual.js

```diff
@@ -0,0 +1,11 @@
+class Component extends React.Component {
+  render() { return null }
+}
+
+Component.propTypes = {
+  children: React.PropTypes.string.isRequired,
```

Comment:
can you add some comments in here like `/* comment */` like the other files so we can make sure those are extracted properly?